### DISCUSSION TITLE
4.2.1

### DISCUSSION THREAD

**Discussion by quantumsheep** [quantumsheep/sshs#78] [Announcements]
## 🐛 Bug fixes 🐛
* Changed the template's default value to let ssh decide which port to use
* Hosts without an `Hostname` directive will show as if the hostname was the host's name (Fixes #77)

**See All the Changes:** [Here's the full list of updates from version 4.2.0 to 4.2.1](https://github.com/quantumsheep/sshs/compare/4.2.0...4.2.1)

<hr /><em>This discussion was created from the release <a href='https://github.com/quantumsheep/sshs/releases/tag/4.2.1'>4.2.1</a>.</em>

**Comment by mchalunderscore** [comment]
woah!!! this has blown up with progress, awesome!! looks like i'll need to clone nixpkgs again... 😅

**Reply by quantumsheep** [reply]
Hey! Yeah I deleted the flake since I didn't know how to update it 😂 I'll be glad if you reimplement it (with GitHub Actions? 👀)
